api/config/testdata/config-2.yaml

```diff
@@ -1,17 +1,24 @@
-Docs:
+docs:
   - label: "Merlin User Guide"
     href: "https://github.com/gojek/merlin/blob/main/docs/getting-started/README.md"
 
-Authorization:
-  Enabled: true
-  KetoServerURL: http://localhost:4466
+applications:
+  - name: Turing
+    description: ML Experimentation System
+    homepage: /turing
+    configuration:
+      aPI: /api/turing/v1
```

Comment:
`aPI` looks strange in lowerCamelCase 😶

Reply:
it should be just `api`, thanks for catching it. But also, case doesn't matter for reading configuration from yaml file, so both `configuration.api` and `Configuration.API` will be mapped to the correct field in the `Config` struct.